.circleci/config.yml

```diff
@@ -128,6 +138,23 @@ aliases:
       name: Waiting for Dockerized request replayer
       command: wait-for request-replayer:80 --timeout=120
 
+  - &STEP_WAIT_TEST_AGENT
+    run:
+      name: Waiting for Dockerized APM Test Agent
+      command: wait-for test-agent:9126 --timeout=120
```

Comment:
Do we really need a 2 minutes timeout ? We're starting to chase unnecessary wait time to reduce ci cost

Reply:
reduced this to 30 seconds